src/components/email-flow/EmailFlowCard.tsx

```diff
@@ -0,0 +1,52 @@
+import { Suspense } from 'react'
+import { useSnapshot } from 'valtio'
+import { useState } from 'preact/hooks'
+import Card from 'components/Card'
+import CardContainer from 'components/proofs/CardContainer'
+import ConnectAccount from 'components/proofs/ConnectAccount'
+import EmailFlowProof from 'components/email-flow/EmailFlowProof'
+import LoadingTitle from 'components/proofs/LoadingTitle'
+import WalletStore from 'stores/WalletStore'
+
+import EmailFlowForm from 'components/email-flow/EmailFlowForm'
+import proofStore from 'stores/ProofStore'
+
+export default function () {
+  const { account } = useSnapshot(WalletStore)
+  const { emailProofsCompleted } = useSnapshot(proofStore)
+  const [domain, setDomain] = useState<string | undefined>()
+
+  function onUpdateDomain({ domain }: { domain?: string }) {
+    setDomain(domain)
```

Comment:
When we're done, we can optimize this function by adding `debounce` so we don't run too many operations when the user is typing, only when the user has finished typing

Reply:
Added a task for this